packages/components-react/projects/react-wrapper/src/provider.tsx

```diff
@@ -12,7 +12,7 @@ type Props = { prefix?: string };
 export const PorscheDesignSystemProvider = ({ prefix = '', ...props }: PropsWithChildren<Props>): JSX.Element => {
   useEffect(() => {
     load({ prefix });
-  }, [prefix]);
+  }, []);
```

Comment:
might fail if it was linted

Reply:
Add Ticket to enable linting in utilities/angular/react projects